resources/views/livewire/custom-field-set-default-values-for-model.blade.php

```diff
@@ -123,10 +124,17 @@ class="form-control"
                                         Unknown field element: {{ $field->element }}
                                     </span>
                                 @endif
+                                        <?php
+                                        $errormessage = $errors->first($field->db_column_name());
+                                        if ($errormessage) {
+                                            $errormessage = preg_replace('/ snipeit /', '', $errormessage);
```

Comment:
Can we pull the number off of the end as well?

Reply:
(We should really probably try to figure out a better way to handle this higher up, since we have to strip off the beginning and the end of them in the asset as well.)

Reply:
Ah, yeah, I'd just copied it from elsewhere. I'll try to track down all the other instances. Or come up with a way to handle it better on the model or something.

Reply:
So, because the attributes names are getting set through the validation message, there's really not a great way to do this "at the top" - we could have an array of replaced attribute names in our language files, but that doesn't make a lot of sense with custom field names. You can also do something with constants and attribute "pretty" names - but this also doesn't make a lot of sense here. 

The closest thing I found that _might_ have worked, was doing this in the validator instance, with a `setAttributeNames` method, but this doesn't work here either because we're adding the rules to the `$rules` array for watson validating. 

So, unfortunately, I just went ahead and added the regex for removing the digits.

Reply:
Thanks @marcusmoore for https://github.com/dwightwatson/validating/blob/7798363f63d05565ccae01ae95c2227519c1739f/src/ValidatingTrait.php#L367
looking into that as an option to use `setAttributeNames()`

Reply:
Alright, I'm dumb and this is actually happening _before_ WatsonValidating runs in `AssetModelsController` - so added `->setAttributeNames()` there. I'll look into adding it up at the model level as well, but this resolves this specific issue.

Reply:
a new screenshot for good measure. 

<img width="903" alt="image" src="https://github.com/user-attachments/assets/7c31372a-69bf-4e24-af55-9f32ecf152b0">